server/game/GameActions/MakeTokenCreatureAction.js

```diff
@@ -6,6 +6,7 @@ class MakeTokenCreatureAction extends PlayerAction {
         this.deployIndex = undefined;
         this.cards = null;
         this.cardLocation = 'deck';
+        this.alwaysSucceed = false;
```

Comment:
what is this for? I think you submitted TheVisibleHandle change to Outnegotiate issue. 

Keep this issue to fix only Outnegotiate.

Reply:
Oh wow, don't know how that happened, sorry. Fixed!